contracts/tests/ocr2.spec.ts

```diff
@@ -223,6 +229,7 @@ describe("ocr2", async () => {
             isSigner: false,
           },
           { pubkey: storeAuthority, isWritable: false, isSigner: false },
+          { pubkey: SYSVAR_INSTRUCTIONS_PUBKEY, isWritable: false, isSigner: false },
```

Comment:
nit: is there a test case where `SYSVAR_INSTRUCTIONS_PUBKEY` isn't passed to the contract?

thinking about the case where the contract is upgraded before the node is upgraded (or vice versa)